content/blog/index.html.haml

```diff
@@ -34,7 +34,7 @@ section: blog
           %a{:href => '/blog/', :tabindex => -1}
             The Jenkins Blog
       %div.app-app-bar__controls.app-mobile-hide
-        %div{:class => page.selected_tag ? "app-button app-tags-picker app-tags-picker--active" : "app-button app-tags-picker"}
```

Comment:
Due to the button style changes this class had to be added to keep the UI as it was before